money-monkey-web/lib/supabase/server.ts

```
@@ -0,0 +1,30 @@
+import { createServerClient } from "@supabase/ssr";
+import { SupabaseClient } from "@supabase/supabase-js";
+import { cookies } from "next/headers";
+
+export async function createClient(): Promise<SupabaseClient> {
+  const cookieStore = cookies();
+
+  return createServerClient(
+    process.env.NEXT_PUBLIC_SUPABASE_URL!,
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)